CHANGELOG.md

```diff
@@ -20,6 +20,9 @@ a release.
 
 ## [Unreleased]
 
+### Fixed
+- Add compatibility with doctrine/orm >= 2.10 (#2272)
```

Comment:
```suggestion
- Restored compatibility with doctrine/orm >= 2.10.2 (#2272).
  Since doctrine/orm 2.10, `Doctrine\ORM\UnitOfWork` relies on SPL object IDs instead of hashes, we need to adapt our codebase in order to be compatible with this change.
  As `Doctrine\ODM\MongoDB\UnitOfWork` from doctrine/mongodb-odm still uses `spl_object_hash()`, all `spl_object_hash()` calls were replaced by `spl_object_id()` to make it work with both ORM and ODM managers.
```